### DISCUSSION TITLE
Timestamps of schemaOptions

### DISCUSSION THREAD

**Discussion by diego-rapoport** [TypedProject/ts-express-decorators#1790] [Q&A]
So I saw that is possible to change the names `createdAt` and `updatedAt`, but couldn't find a clear example of it. From the lib itself it says that I need to change `timestamps.createdAt` inside the class, but I couldn't figure it out how.
Can someone help?

**Comment by Romakita** [comment]
Maybe by using the decorator:

@Model({schemaOptions: {timestamps: {createdAt}})

**Reply by diego-rapoport** [reply]
Thank you very much! Will test it soon,
